tests/dbshell/test_oracle.py

```diff
@@ -0,0 +1,34 @@
+from unittest import mock, skipUnless
+
+from django.db import connection
+from django.db.backends.oracle.client import DatabaseClient
+from django.test import SimpleTestCase
+
+
+@skipUnless(connection.vendor == 'oracle', 'Oracle tests')
+class OracleDbshellCommandTestCase(SimpleTestCase):
+    def _run_dbshell(self, rlwrap=False):
+        """Run runshell command and capture list of args."""
+        client = DatabaseClient(connection)
```

Comment:
Put this line above `self.subprocess_args = None` (would remove the need for an extra blank line, I think)?

Reply:
Agreed.